Tensile/Contractions.py

```diff
@@ -380,9 +380,13 @@ def CompoundPredicates(cls, state, problemType):
             rv += [cls("KernelLanguageCompatible", value=state["KernelLanguage"])]
 
         if ('GlobalSplitU' in state) and (state['GlobalSplitU'] > 1):
-            if ('_GlobalAccumulation' not in state) or (state['_GlobalAccumulation'] != 'MultipleBuffer'):
+            if ('_GlobalAccumulation' not in state) or (state['_GlobalAccumulation'] == 'SingleBuffer'):
```

Comment:
Isn't None case covered by ('_GlobalAccumulation' not in state) ?

Reply:
> Isn't None case covered by ('_GlobalAccumulation' not in state) ?

Not really.
In assignDerivedParameters (in SolutionStruct.py), 

    if ("_GlobalAccumulation" not in state) or ("_WorkspaceSizePerElemC" not in state):
      state["_GlobalAccumulation"] = None

we set tate["_GlobalAccumulation"] = None first.
If GSU>1 and SingleBuffer and ComputeType == DestType, state["_GlobalAccumulation"]  remains None.

Reply:
GSU>1 and SingleBuffer and ComputeType == DestType case, we do not need to call postGSU function.
state["_GlobalAccumulation"] = None means no postGSU call.

Here, we need to check if GSU is deterministic (Multi) or non-determinstic (Single).
We have 2 possibilities state["_GlobalAccumulation"] = 'SingleBuffer' or None depending on dest type.

Reply:
Thanks for the explanation, I misread this part. Updated it to restore it to the original check since this only applies to GSU, and I have a separate check for deterministic stream-k.